test/e2e/specs/customer_reader.go

```diff
@@ -0,0 +1,88 @@
+package specs
+
+import (
+	"errors"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	kerrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+
+	"github.com/openshift/openshift-azure/pkg/util/randomstring"
+	"github.com/openshift/openshift-azure/test/clients/openshift"
+)
+
+var _ = Describe("Openshift on Azure customer-reader e2e tests [CustomerAdmin]", func() {
+	var (
+		cli       *openshift.Client
+		readercli *openshift.Client
+	)
+
+	BeforeEach(func() {
+		var err error
+		cli, err = openshift.NewSystemAdminClient()
```

Comment:
Likewise. This project creation/deletion should be something done by an end user.

Reply:
change to `openshift.NewEndUserClient()`